doc/segmentation_output.rst

```diff
@@ -0,0 +1,12 @@
+Segmentation Output
+-------------------------
+
+Segmentation outputs a mask (`iris.cube.Cube` and in the future `xarray.DataArray`) with the same dimensions as the input field, where each segmented area has the same ID as its corresponding feature.
```

Comment:
I think it's worth clarifiying that the ID is the `feature` column in the dataframe.

Reply:
Good point. Done!